test/run

```diff
@@ -16,7 +16,18 @@ test -n "${IMAGE_NAME-}" || { echo 'make sure $IMAGE_NAME is defined' && false ;
 test -n "${VERSION-}" || { echo 'make sure $VERSION is defined' && false; }
 test -n "${OS-}" || { echo 'make sure $OS is defined' && false; }
 
-test_exit=1
+test_short_summary=''
+TESTSUITE_RESULT=0
+
+TEST_LIST="\
+run_container_creation_tests
```

Comment:
From my point of view is missing check_result inside run_container_creation_tests

Reply:
Could point I have added `check_result` inside `try_image_invalid_combinations`.